D-lama-service/Data/ProjectEntities/Project.cs

```diff
@@ -42,7 +45,17 @@ public Project(string name, string description)
             Name = name;
             Description = description;
             UpdateDate = DateTime.UtcNow;
-            IsReady = false;
+            DataType = "text";
```

Comment:
Maybe it is easier to use an Enum here, as you can then make comparison using the Enum later on:

if(project.DataType == ProjectType.Text){...}, so you don't have the coupling of the explicit string value.

just as an example.

Reply:
That's a great advice. I forgot about enums completely!